pulp_nightly/Containerfile

```diff
@@ -1,4 +1,5 @@
-FROM pulp/pulp-ci-centos:latest
+ARG FROM_TAG="latest"
+FROM pulp/pulp-ci-centos:${FROM_TAG}
```

Comment:
:heart:

Reply:
This looks like the cleanest approach to me so far.
Maybe this is also the way to forward with images for different python versions.